### DISCUSSION TITLE
Exercise Section 01: Large Data set leading to poor model performance

### DISCUSSION THREAD

**Discussion by adesouza1** [mrdbourke/pytorch-deep-learning#526] [Q&A]
First of all: Great Course I'm learning a lot. I was messing around with the model and changing variables to get a better intuitive feel for how the LinearRegressionModel works and I ran into an interesting situation where as I increase the number of data points I reached a point where the model no longer learned. The loss function fails to decrease so I suspect it has to do with some to do with it but I haven't come up with a good reason for why. (I changed end = 100 so len(X_train) = 4000. It gets stuck like this regardless of the number of epochs I run.

![download](https://github.com/mrdbourke/pytorch-deep-learning/assets/32052596/82768ca0-9aec-4bc9-ba14-790087310c9f)
![download](https://github.com/mrdbourke/pytorch-deep-learning/assets/32052596/a8cc50f8-8d17-45c6-a904-0cca0e8fb8e4)

**Comment by adesouza1** [comment]
Partially answering my own question here but I discovered that by decreasing the learning rate below what we used in the example allowed for a much better fit.
